modules/ROOT/pages/cloudhub-fabric.adoc

```diff
@@ -50,7 +51,7 @@ To learn more about how to work with persistent queues in your application, see
 
 You can enable and disable either or both features of CloudHub Fabric in one of two ways:
 
-* When deploying an application to CloudHub for the first time using the Runtime Manager console
+* When you deploy an application to CloudHub for the first time using the Runtime Manager console, or
```

Comment:
I'd leaver out the ", or" since you state clearly in the lead-in "one of two ways." Not sure we have a specific style ruling.